### ISSUE TITLE
Reproduce training stage

### ISSUE THREAD

**Issue by DadEap** [herohuyongtao/deeptag-pytorch#9]
Hello, 

I would like to reproduce your training steps to learn your models, but I could not find a file to do it. 
Could you tell me how to reproduce this step ? Is this included in this project?

Thanks for your work
Best regards

**Comment by SujithChristopher** [comment]
+1, I would also like to reproduce the training stage. Thanks.

**Comment by MoradLaglil** [comment]
Hello everyone,

I am currently working on training a model, but I am facing difficulties in implementing the loss functions due to some ambiguity in the description. I was wondering if anyone could help me reproduce the training stage and provide some guidance. Your assistance would be highly appreciated.

Thank you in advance for your support.

**Comment by qifang-robotics** [comment]
> n

Hi morad, I am also working on the reproduction of training. How is your progress in it? Could you share some training code?

**Comment by qifang804** [comment]
> Hello everyone,
> 
> I am currently working on training a model, but I am facing difficulties in implementing the loss functions due to some ambiguity in the description. I was wondering if anyone could help me reproduce the training stage and provide some guidance. Your assistance would be highly appreciated.
> 
> Thank you in advance for your support.

@moradMIRO Hi morad, I am also working on the reproduction of training. How is your progress in it? Could you share some training code?
Also, if anyone has reproduced the training stage, could you share them? Deeply appreciated.
@DadEap @SujithChristopher @herohuyongtao @sskzzm
